lib/iris/mesh/components.py

```diff
@@ -2808,6 +2752,31 @@ def location(self):
     def axis(self):
         return self._metadata_manager.axis
 
+    @property
+    def points(self):
+        """The coordinate points values as a NumPy array."""
+        if self.timestamp < self._mesh.timestamp or self.timestamp is None:
+            self._load_points_and_bounds()
+        return self._values
+
+    @points.setter
+    def points(self, value):
+        if value:
+            msg = "Cannot set 'points' on a MeshCoord."
+            raise ValueError(msg)
+
+    @property
+    def bounds(self):
+        if self.timestamp < self._mesh.timestamp or self.timestamp is None:
+            self._load_points_and_bounds()
+        return self.bounds
+
+    @bounds.setter
+    def bounds(self, value):
+        if value:
+            msg = "Cannot set 'bounds' on a MeshCoord."
+            raise ValueError(msg)
```

Comment:
Gonna need some more nuance somewhere. Calling `super.__init__()` runs the below line:

https://github.com/SciTools/iris/blob/7b32abe214330d294b7f540c8774a2fca668781c/lib/iris/coords.py#L1509

This is unrelated to WHEN we call `super().__init__()` (a separate thread), because even if we change that we will still need to call it somewhere during `MeshCoord.__init__()`.